Scripts/package.py

```diff
@@ -127,21 +127,23 @@ def zipify_path(fpath):
         """Zip entries always use '/' as the path separator."""
         return fpath.replace(os.path.sep, '/')
 
-    def run_publish(self, project):
+    def run_publish(self, project, framework = None):
         env = dict(os.environ)
         env["RUNTIME_IDENTIFIER"] = self.target
         flush("   + Publishing " + project)
         remaining = 3
         exitStatus = 1
         while 0 < remaining and exitStatus != 0:
             remaining -= 1
+            otherArgs = []
+            if framework:
+                otherArgs += ["-f", framework]
             exitStatus = subprocess.call(["dotnet", "publish", path.join(SOURCE_DIRECTORY, project, project + ".csproj"),
                 "--nologo",
-                "-f", "net6.0",
                 "-o", self.buildDirectory,
                 "-r", self.target,
                 "--self-contained",
-                "-c", "Release"], env=env)
+                "-c", "Release"] + otherArgs, env=env)
```

Comment:
```suggestion
                "-c", "Release", *framework_args], env=env)
```

Reply:
Ah thank you, I thought I recalled something like that. If the CI has to be rerun anyway I'll apply this :)